unity-renderer/Assets/DCLPlugins/ECS7/Systems/ECSSystemsController.cs

```diff
@@ -73,8 +78,8 @@ public ECSSystemsController(ECS7System componentWriteSystem, SystemsContext cont
 
         lateUpdateSystems = new ECS7System[]
         {
-            ECSCameraEntitySystem.CreateSystem(context.componentWriter),
-            ECSPlayerTransformSystem.CreateSystem(context.componentWriter)
+            cameraEntitySystem.Update,
```

Comment:
For the future: Why do we need to allocate a delegate from the method group if we could have a contract for systems like:
```
interface IECSSystem {
void Update();
}
```

Reply:
Yeah I see what you mean, I'm not sure why this was originally implemented like it is now but it would be more maintainable/easy-to-understand using a simple interface like the one you suggest.
I'll discuss a possible mini refactor with @pbosio to change that in another PR.